python/tests/embedded/test_ceiling.py

```diff
@@ -26,9 +26,8 @@ def sample_raw() -> element.Element:
     return element.Element.from_string(data)
 
 
-@pytest.fixture
-def bad_measurements() -> element.Element:
-    data = """
+BAD_DATA_XML = [
```

Comment:
As discussed IRL, could you add a comment saying what's wrong with each of the blocks of bad XML?